### PR TITLE
Ebnt 141 uwsgi

### PR DESCRIPTION
**Pull request by mike0sv**
added uwsgi and nginx for flask based containers

### CODE REVIEW

File: src/ebonite/ext/flask/build_templates/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,31 @@
+FROM python:3.6
```
Comment (i-a-andrianov) [inline src/ebonite/ext/flask/build_templates/Dockerfile]: is this file really needed?

Comment (mike0sv) [reply src/ebonite/ext/flask/build_templates/Dockerfile]: no ;)
